File: 07_gpu_target/problems/00_target_offload/target_offload.f90
program target_offload
  use omp_lib
  ! TODO: 下の print を !$omp target ... !$omp end target で囲み, デバイス(GPU)上で実行させよ.
  print "(a)", "hello from the device"
  ! TODO: 上で始めた target 領域を閉じる (!$omp end target).
end program target_offload

File: 07_gpu_target/problems/00_target_offload/target_offload.cpp
#include <cstdio>
#include <omp.h>

int main() {
  // TODO: 下のブロックの直前に #pragma omp target を1行追加し, printf をデバイス(GPU)上で実行させよ.
  {
    printf("hello from the device\n");
  }
  return 0;
}
